tests/009-alpns.cpp

```diff
@@ -28,31 +27,18 @@ namespace oxen::quic::test
 
         Network test_net{};
 
-        SECTION("Do not get ALPN before negotiated")
```

Comment:
this test case still seems valid

Reply:
yup but lets move it to 008 and rename it to "connections" then make a test case on "do not access values before connection established"